plugins-jwt-auth-and-shield/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "auth",
+  "dependencies": {
+    "bcryptjs": "2.4.3",
+    "jsonwebtoken": "8.5.1",
+    "nexus": "0.21.0-next.14",
+    "nexus-plugin-jwt-auth": "1.0.0",
+    "nexus-plugin-prisma": "0.7.0",
+    "nexus-plugin-shield": "^0.1.2"
```

Comment:
To follow the convention this example should be called `plugins-jwt-auth-and-shield-and-prisma`.